src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
+function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    // Add login logic here
```

Comment:
**suggestion**: Implement form validation to ensure that the username and password fields are not empty before submission. This will improve user experience and prevent unnecessary form submissions.